### DISCUSSION TITLE
Issues with an X-Pander 3 and an Easy Flash 3?

### DISCUSSION THREAD

**Discussion by GorrillaRibs** [frntc/RAD#42] [Q&A]
I've got an NTSC C64 with the above things attached, but no matter what timings I choose I can't seem to get it to run stable - the RAD works just fine & passes a memory test without the X-Pander, and the Easy Flash also works fine without the X-Pander and on it without the RAD. I did get the Rad to pass a ram test on the X-Pander once (with timings of WAIT_ENABLE_RW_ADDRLATCH=505 and WAIT_ENABLE_DATA_WRITING=735, but couldn't replicate that result). Default timings work just fine without the X-Pander in the mix.

What could be the issue here? Is there potentially a fault with my C64/some conflict with the Easy Flash? If anyone's got any idea for a solution would love to know (otherwise I'll just cave and finally get order jiffydos on a real ROM so I don't have to rely on the easy flash). Thanks!

**Comment by frntc** [comment]
I don't have an X-Pander, but some guys tested other cartridge port expanders and with some adjusted timings they seem to work fine (the slot in the expander might make a difference). In general these expanders seem to have an impact on signals and timings.

Note that in general the EF3 and RAD (or any other combination of such cartridges) conflict on the bus! You might be lucky and the EF3 in kernal replacement mode (in case it's really just doing that, haven't checked) does not touch IO1/2, Dx, GAME/EXROM etc. at the wrong time, but I certainly recommend to not running the two in parallel!

**Reply by GorrillaRibs** [reply]
Thanks for the reply! I did indeed get the RAD working by itself on the X-Pander by fiddling with the timings some more. The EF3 not being compatible is a bummer, but isn't the end of the world - I'll see if I can fiddle with enabling/disabling lines (there's a switch for each line on the X-Pander, so I can test what works and doesn't with the EF3), I'll report back if I figure it out

**Reply by DirkS-71** [reply]
Would be helpful for the community if you share the slot and timings you were successful with.
